Library/Homebrew/os/mac.rb

```diff
@@ -39,29 +37,28 @@ def full_version
     end
 
     sig { params(version: String).void }
-    def full_version=(version)
+    def self.full_version=(version)
       @full_version = Version.new(version.chomp)
       @version = nil
     end
 
     sig { returns(::Version) }
-    def latest_sdk_version
+    def self.latest_sdk_version
       # TODO: bump version when new Xcode macOS SDK is released
       # NOTE: We only track the major version of the SDK.
       ::Version.new("13")
     end
-    private :latest_sdk_version
```

Comment:
This was useless, since it only affected the instance method, which was already private. For example: https://github.com/Homebrew/brew/blob/e98bfdd/Library/Homebrew/os/mac/xcode.rb#L69